parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -174,7 +173,7 @@ installKubeletKubectlAndKubeProxy() {
             extractKubeBinaries ${KUBERNETES_VERSION} ${KUBE_BINARY_URL}
         else
             if [[ "$CONTAINER_RUNTIME" == "containerd" ]]; then
-                extractHyperkube "ctr"
+                extractHyperkube "ctr" #TODO: use crictl if installed though in theory after 1.19.x hyperkube is no longer used
```

Comment:
we already stop baking hyperkube for 1.19. I think we can remove this now

Reply:
I will remove the TODO comment in this PR and remove entire hyperkube code for containerd in a separate PR

Reply:
added an ado item to track removal of hyperkube code for containerd